### DISCUSSION TITLE
How to become a maintainer in hactoberfest?

### DISCUSSION THREAD

**Discussion by Vaibhav1808** [PEC-CSS/Open-Source#426] [Q&A] [answered]
How do we participate in hactoberfest as maintainer?

**Comment by Manobal-Singh-Bagady** [comment, marked as answer]
here's the answer:  https://www.loginradius.com/blog/engineering/hacktoberfest-participation-as-maintainer/

**Comment by Manobal-Singh-Bagady** [comment]
There are also rewards for maintainers:

![image](https://user-images.githubusercontent.com/96862518/196043810-4db6a9a9-2199-496f-8025-1028a772b08a.png)
